db/utility.js

```diff
@@ -0,0 +1,17 @@
+exports.formatArticles = (articleArray, commentArray) => {
+  const commentCount = {};
+  commentArray.forEach((comment) => {
+    commentCount[comment.article_id] = comment.count;
+  });
+
+  return articleArray.map((object) => {
```

Comment:
If you wanted to, you could do this logic (naming column comment_count and setting default values) in your db query...
This may be of interest: https://www.w3schools.com/sql/sql_ref_as.asp

Reply:
That looks useful! Thanks!